src/main/resources/db/migration/V1_86__juror_audit-update.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE juror_mod.juror
+    add column last_modified_by varchar(20);
```

Comment:
this will be difficult to derive for migration - looks like it can allow nulls so I assume it won't cause any problem if migrated data doesn't populate it?

Reply:
user_edtq  is  never updated in code.

My proposal would be to remove this field in favour  for last_modified_by? (or vis versa with the addition of the @LastModifiedBy annotation)

Reply:
Code updated to remove last_modified_by field and instead add this into the rev_info table